src/d3a/models/strategy/update_frequency.py

```diff
@@ -151,6 +151,11 @@ def calculate_initial_sell_rate(self, current_time_h):
         else:
             raise ValueError("Initial rate option should be one of the InitialRateOptions.")
 
+    def _price_update_interval(self):
```

Comment:
On the previous comment, I mentioned that it would be great for this to be a property. What I meant was not just renaming this method, but to convert it to a property by adding the @property decorator, so that you do not have to call the function like self._price_update_interval() (once it is converted to a property, the brackets () can be removed from the method calls)